content/html/concepts/semantic-html/terms/meter/meter.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '<meter>'
+Description: 'Represents either a fractional value or a scalar value within a known range.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<meter>` semantic tag is used to represent a fractional or a scalar value within a known range. This form of representation is known as a gauge.
+
+> **Note**: The `<meter>` tag should not be used to indicate progress (as in a progress bar).
```

Comment:
[Proofreading] I just published an entry on the `<progress>` element, so consider adding the link to this statement just as I did here:
```suggestion
> **Note**: The `<meter>` tag should not be used to indicate [progress (as in a progress bar)](https://www.codecademy.com/resources/docs/html/semantic-html/progress).
```